.github/workflows/delete-gcp-resources.yml

```diff
@@ -22,6 +22,7 @@ jobs:
           workload_identity_provider: 'projects/143793276228/locations/global/workloadIdentityPools/github-actions/providers/github-oidc'
           service_account: 'github-service-account@zealous-zebra.iam.gserviceaccount.com'
           token_format: 'access_token'
+          create_credentials_file: false
```

Comment:
```suggestion
```

I tested this as I recall using this approach failed before. I once included this as I saw a GHA's warning, but it doesn't work as expected.

Reply:
Ok, we should probably use the checkout action to silence the warning

Reply:
Fixed by:
https://github.com/ZcashFoundation/zebra/pull/5082/files#diff-934c576d4f57156181f9078afdfef6c33b80cca73f5825f6f06ad72ed018a4efR25